app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java

```diff
@@ -242,7 +246,7 @@ boolean descriptionInfo(View view, MotionEvent motionEvent) {
     }
 
     private void setLicenseSummary(String license) {
-        licenseSummaryView.setText(getString(R.string.share_license_summary, getString(Utils.licenseNameFor(license))));
+        licenseSummaryView.setHtmlText(getString(R.string.share_license_summary, getString(Utils.licenseNameFor(license))));
     }
```

Comment:
Upload crashes due to these changes. It seems to me like these changes are related with another PR that you have submitted #1086 . To remove this confusion it is always better to work on separate branches instead of master, and focusing only one task at a time.